block/submit.go

```diff
@@ -110,6 +110,8 @@ func SubmitLoopInner(
 				UpdateBatchSubmissionGauges(pending, unsubmittedBlocksNum(), batchSkewTime())
 
 				if done || nothingToSubmit || (lastSubmissionIsRecent && maxDataNotExceeded) {
+					pendingBytes.Store(pending)
+					ticker.Reset(maxBatchSubmitTime)
```

Comment:
I don't understand this. can you please explain? thanks

Reply:
moved after loop and documented.